### DISCUSSION TITLE
Was using gitlab container registry to store CNAB bundle. Used to work, now  getting "Invalid tag: missing manifest digest"

### DISCUSSION THREAD

**Discussion by christianbeland** [getporter/porter#1544] [Q&A]
I was using gitlab container registry to store CNAB bundle for a while.

It used to work, but since I updated gitlab, I am now getting "Invalid tag: missing manifest digest" on gitlab  container registry UI for CNAB bundle tags. Porter cannot push those bundle anymore (unexpected response: 400 Bad Request).

I cannot pull then either (400 Bad Request).

Anyone is currently successfully using gitlab container registry with porter at the moment?

**Comment by vdice** [comment]
Interesting.  I wonder if there are release notes for the Gitlab update that might hint on any changes that would affect bundle push/pull.  Are you still able to push/pull the same bundles to another container registry successfully?

**Reply by christianbeland** [reply]
I will try harbor... but wanted to know if anybody else was using gitlab sucessfully. I would have rather stayed with gitlab...

I saw https://gitlab.com/gitlab-org/container-registry/-/issues/266: "We'll be disabling support for schema 1 manifest on January 22, 2021. For this reason, the database metadata import tool should ignore schema 1 manifests." Not sure if it could have an impact.
